range_queries/heavy_light_decomposition.cpp

```diff
@@ -0,0 +1,446 @@
+/*
+ * @file
+ * @Simple Heavy-Light Decomposition implementation
+ * Author: Aniruthan R
```

Comment:
You can use Markdown format, and it will render properly in the website documentation.
```suggestion
 * @author: [Aniruthan R](https://github.com/aneee004)
```